.isort.cfg

```diff
@@ -0,0 +1,9 @@
+[settings]
+line_length = 120
+skip = migrations
+known_edx =
+known_django = django
+known_first_party =
+    workbench
+    sample_xblocks
+sections = FUTURE,STDLIB,THIRDPARTY,DJANGO,DJANGOAPP,EDX,FIRSTPARTY,LOCALFOLDER
```

Comment:
We should probably standardize on an order for these pretty soon; I think the cookiecutter and edx-platform still conflict (this looks like the cookiecutter order).

Reply:
Agreed. This is the cookiecutter version, which I tend to use as the canonical way of doing things for these app / xblock repos (for better or worse, I just want things to be as consistent as they can be).

Reply:
Ugh - not looking forward to going back into edx-platform and running isorts again....